context/app/static/js/pages/CellsAPIDemo/DatasetsTable.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Table from '@material-ui/core/Table';
+import TableHead from '@material-ui/core/TableHead';
+import TableBody from '@material-ui/core/TableBody';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+
+import { LightBlueLink } from 'js/shared-styles/Links';
+
+const columns = [
+  { id: 'hubmap_id', label: 'HuBMAP ID' },
+  { id: 'origin_sample.mapped_organ', label: 'Organ' },
+  { id: 'mapped_data_types', label: 'Mapped Data Types' },
+  { id: 'donor.mapped_metadata.age_value', label: 'Donor Age' },
+  { id: 'donor.mapped_metadata.body_mass_index_value', label: 'Donor BMI' },
+  { id: 'donor.mapped_metadata.sex', label: 'Donor Sex' },
+  { id: 'donor.mapped_metadata.race', label: 'Donor Race' },
+
+  { id: 'last_modified_timestamp', label: 'Last Modified' },
+];
+
+function DonorMetadataCell({ unit, value }) {
```

Comment:
```suggestion
function UnitValueCell({ unit, value }) {
```
There are donor fields that don't have units, and there are dataset fields that do.